srcalc/src/main/java/gov/va/med/srcalc/service/AdminService.java

```diff
@@ -11,23 +13,36 @@
 public interface AdminService
 {
     /**
-     * Returns all Variables for editing purposes.
+     * Returns all Variables in the database.
+     * @return a list, in display name order
      */
     public List<AbstractVariable> getAllVariables();
     
     /**
-     * Returns the Variable with the given display name.
+     * Returns all VariableGroups in the database..
+     * @return an ImmutableCollection, in arbitrary order
+     */
+    public ImmutableCollection<VariableGroup> getAllVariableGroups();
+    
+    /**
+     * Returns the Variable with the given display name for editing. Note that
+     * the returned object must be given back to {@link
+     * #updateVariable(AbstractVariable)} to persist any changes.
      * @throws InvalidIdentifierException if no such Variable exists
      */
     public AbstractVariable getVariable(final String key)
         throws InvalidIdentifierException;
     
     /**
-     * Updates a Variable with the given command object. See the package javadocs
-     * for why we use a command object.
-     * @param command a command object containing the edits to make
-     * @throws InvalidIdentifierException if no such Variable exists
+     * Updates the given variable in the persistent store.
+     * @param variable the variable to update. The object must have been
```

Comment:
I didn't link it due to the advice in [this Javadoc style guide](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#styleguide). They say to only link if a user might actually want to click on it, which in this case is already handled because they can just click on the parameter type.